src/Colors.stories.mdx

```diff
@@ -0,0 +1,49 @@
+<!-- Colors.stories.mdx -->
+
+import { Meta, ColorPalette, ColorItem } from '@storybook/addon-docs';
+import './scss/colors.scss';
+
+<Meta title="Colors" />
+
+<ColorPalette>
+  <ColorItem title="Primary" subtitle="LighterBordeaux" colors={{ Background: '#eee8ea' }} />
```

Comment:
Is there maybe a way to get the actual value from the `scss` file here?
No idea if it's possible, but it would be real cool

Reply:
Had the same idea. Will investigate this.